### PR TITLE
update: Bump `dbt` to 0.19.1

### PR DESCRIPTION
**Pull request by mrshu**
* Bump the version of `dbt` to 0.19.1

* Bump the version of `pyathena` to `2.2.0`

Signed-off-by: mr.Shu <mr@shu.io>

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -59,7 +59,7 @@ def _dbt_presto_version() -> str:
         ]
     },
     install_requires=[
-        "dbt-core==0.19.0",
-        "pyathena==2.1.1",
+        "dbt-core==0.19.1",
+        "pyathena==2.2.0",
```
Comment (Tomme) [inline setup.py:63]: Upgrading to `pyathena` 2.2.0 will likely cause an issue within `connections.py`, specifically [L29](https://github.com/Tomme/dbt-athena/blob/70a91f667c42741b58aa650e47bdf848765df089/dbt/adapters/athena/connections.py#L29).

If you set `work_group` to default to `None` it should then be compatible.

This is due to the following PR [#221](https://github.com/laughingman7743/PyAthena/pull/221) within `pyathena`

**Review by Tomme — CHANGES_REQUESTED** [review]
Thanks for the contribution! If you can resolve the comment I have then rebase your branch to `master` I will run the `make run_tests` again and merge your changing in 👍

**Comment by mrshu** [comment]
Thanks @Tomme, I believe your comment should be resolved. 

I'll see if I can get the tests to run myself as well but would appreciate if you could try them out.

Thanks!

**Comment by danfran** [comment]
are there still any issues with this PR?

**Pull request merged**